rust/src/transformations/make_stable_lazyframe/filter/test.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{
+    domains::{AtomDomain, LazyFrameDomain, Margin, OptionDomain, SeriesDomain},
+    metrics::SymmetricDistance,
+    transformations::make_stable_lazyframe,
+};
+
+use super::*;
+
+#[test]
+fn test_filter() -> Fallible<()> {
+    let lf = df!("chunk_2_null" => [[Some(1i64); 500], [None; 500]].concat())?.lazy();
```

Comment:
Is there anything that 500 exercises that 5 wouldn't? Obviously this is a trivial cost, but tests with larger data might be that much harder to debug.